1-click.go

```diff
@@ -0,0 +1,52 @@
+package godo
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const oneClickBasePath = "/v2/1-click"
+
+// OneClickService is an interface for interacting with 1-clicks with the
+// DigitalOcean API.
+// See: https://developers.digitalocean.com/documentation/v2#vpcs
```

Comment:
This link is not correct. There doesn't seem to be any public documentation for this endpoint.

Reply:
_facepalm_ good catch. there isn't any public documentation for this endpoint, yet. how do I go about adding that?